Makefile

```diff
@@ -45,6 +45,17 @@ docker_latest:
 	docker build -t quay.io/pganalyze/collector:latest .
 	docker push quay.io/pganalyze/collector:latest
```

Comment:
The PR description says "rename" but this code is not deleted--is this still useful?

Reply:
Whoops, good point.